examples/pimlico/src/account.js

```diff
@@ -21,7 +21,6 @@ export const getAccount = async (type) => {
             const simpleAccount = await toSimpleSmartAccount({
                 owner,
                 client,
-                factoryAddress: "0x9406Cc6185a346906296840746125a0E44976454",
```

Comment:
This isn't neeeded, and the permissionless library is able to default the correct address depending on v06 vs v07